scenarios/multi_object_tracking/01_training_introduction.ipynb

```diff
@@ -0,0 +1,820 @@
+{
```

Comment:
<p>This section is hard to read. Can we simplify and just say that we train the model on our dataset, starting with a model which was pre-trained on ImageNet (or pre-trained on pedestrian datasets). Details such as the Adam algorithm, etc, either should be at a later point (after the high-level intro) or actually fully removed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>